src/mpc-hc/MainFrm.cpp

```diff
@@ -11769,7 +11769,11 @@ void CMainFrame::OpenCreateGraphObject(OpenMediaData* pOMD)
         m_pGB = DEBUG_NEW CFGManagerDVD(_T("CFGManagerDVD"), nullptr, m_pVideoWnd->m_hWnd);
 
         if (m_bUseSmartSeek) {
-            m_pGB_preview = DEBUG_NEW CFGManagerDVD(L"CFGManagerDVD", nullptr, m_wndPreView.GetVideoHWND(), true);
+            CString drive = pOpenDVDData->title.Left(2);
+            UINT type = GetDriveType(drive);
+            if (type != DRIVE_CDROM || !IsDriveVirtual(drive)) { //no preview seeking for spinning disks

```

Comment:
@adipose: If I'm not wrong it should be `IsDriveVirtual(drive)` and not `!IsDriveVirtual(drive)`